src/main/java/org/example/homework4/ProductCostCalculator.java

```diff
@@ -0,0 +1,32 @@
+package org.example.homework4;
+
+public class ProductCostCalculator {
+    public static void main(String[] args) {
+        // price of the product
+        double unitPrice = 10.0;
+        // Quantity
+        int quantity = 100;
+
+        double totalCost = calculateTotalCost(unitPrice, quantity);
+        System.out.println("Total cost of the product: " + totalCost);
+    }
+    public static double calculateTotalCost (double unitPrice, int quant) {
```

Comment:
I do not like cutting variable names like this: `quant`. Why do you think `quantity` is too long?